packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -267,21 +267,35 @@ const DMT5TradeModal = ({
                     </a>
                 </div>
             </div>
+            <Text
+                align='center'
+                as='div'
+                className='cfd-trade-modal__download-center-text'
+                size={isMobile() ? 'xxxs' : 'xxs'}
+                weight='bold'
+            >
+                {localize('Download Deriv MT5 on your phone to trade with the Deriv MT5 Derived account')}
```

Comment:
@matin-deriv 
This should be based on the selected account.right?
'Download Deriv MT5 on your phone to trade with the Deriv MT5 `Derived/Financial `account'

Reply:
Thanks for pointing out, i did not notice that we have 10 different kind of texts for each of the different platforms and account types.